src/app/organization/organization.component.html

```diff
@@ -14,12 +14,21 @@ <h4> Download </h4>
     <div class="container-fluid">
       <div class="row action-wrapper">
         <div class="col-2"></div>
-        <button class="col-lg-3"> IMPORT </button>
+        <input type="file" class="d-none" accept=".csv" #bulkUploader (change)="onFileUpload($event)">
+        <button class="col-lg-3" (click)="bulkUploader.click()"> IMPORT </button>
```

Comment:
If it's (click) why do you need to call bulluploader.click again?

Reply:
```<Input type="file">``` looks like this:

![image](https://user-images.githubusercontent.com/37747169/49682503-bc0d5f00-face-11e8-95be-9e92ba655e56.png)


It was breaking the UI, so instead I attached **_import_** button to invisible inputs **_browse_** button.
Now it looks like 'Import' button is uploading files.

<img src="https://user-images.githubusercontent.com/37747169/49682469-50c38d00-face-11e8-8062-e7c0b5ed8f97.png" width=50%>

Reply:
I would recommend you can look for better ways to do this, on (click) again a .click is not at all optimal

Reply:
```html
        <label class="col-lg-3 main-button text-center">
            <input type="file" class="d-none"/>
            IMPORT
        </label>
```

Wrapped ```html <input type="file"> ``` in label tag.
Functionality & UI is same. :smile: